core-devel/gcc/02-compiler/defines

```diff
@@ -1,6 +1,6 @@
 PKGNAME=gcc
 PKGSEC=devel
-PKGDEP="binutils gcc-runtime gmp isl mpc mpfr texinfo zlib zstd"
+PKGDEP="binutils gcc-runtime gmp isl mpc mpfr texinfo zlib"
```

Comment:
This change should have been dropped. It would have depended on zstd anyways (as it was pre-installed in BuildKit).

Move this change to `defines.stage2`.